src/transaction/flashback_cl.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+
+/*
+ * flashback_cl.h
+ *
+ */
+
+#ifndef _FLASHBACK_CL_H_
+#define _FLASHBACK_CL_H_
+
+#include <unordered_map>
+#include "dynamic_array.h"
+#include "storage_common.h"
+#include "log_lsa.hpp"
+
+#define FLASHBACK_FIND_SUMMARY_ENTRY(trid, summary_entry, summary_info) \
```

Comment:
IMHO, it would be nice to make the order of parameters 'in in out'.

But, this is not a strong request. So, you can decide whether or not to change it.